modules/ingest-common/src/test/resources/rest-api-spec/test/ingest/50_on_failure.yaml

```diff
@@ -19,6 +19,11 @@
                   "target_field" : "date",
                   "formats" : ["yyyy"]
                 }
+              },
```

Comment:
test verifies that `uppercase` processor is not executed after the `date` processor failed